### PR TITLE
Update help/faq text for v4

### PR DESCRIPTION
**Pull request by rileyhgrant**
Updates the help/faq texts with the updates for v4, per [this google doc](https://docs.google.com/document/d/1DLUWwVFTpkuyQEAb0Ok5Go4EoMh55XPQtxg5JZNWsI4/edit#heading=h.ppf3upmzfts4)

### CODE REVIEW

**Comment by rileyhgrant** [comment]
Hiya @phildarnowsky-broad, per standup, I'm tagging you in for a review on this one so we can (hopefully) get this merged sooner rather than later ahead of deploying a v4 demo.

This PR is almost entirely `.md` content changes, with the exception of a few more involved help texts that are react components that still mostly consist of copy.

File: browser/help/faq/technical-details/what-genetic-ancestry-groups-are-represented-in-the-gnomad-data.tsx [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+  }
+`
+
+const SampleCountTables = () => (
+  <div>
+    <div>
+      <h4>gnomAD v4</h4>
+      <p>
+        For current stats on genetic ancestry groups present in v4, please see the{' '}
+        {/* @ts-expect-error TS(2745) FIXME: This JSX tag's 'children' prop expects type 'never... Remove this comment to see the full error message */}
```
Comment (phildarnowsky-broad) [inline browser/help/faq/technical-details/what-genetic-ancestry-groups-are-represented-in-the-gnomad-data.tsx:30]: General comment here, no action required, just venting: it's a bummer having to add new `@ts-expect-error`s, I'm looking forward to having the breathing room to work on overhauling GBTK again.

File: browser/help/faq/technical-details/what-genetic-ancestry-groups-are-represented-in-the-gnomad-data.tsx [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+                <th scope="col">non-v2</th>
+              </tr>
+            </thead>
+            <tbody>
+              {['afr', 'ami', 'amr', 'asj', 'eas', 'fin', 'mid', 'nfe', 'sas', 'oth'].map(
+                (popId) => (
+                  <tr key={popId}>
+                    {/* @ts-expect-error TS(7053) FIXME: Element implicitly has an 'any' type because expre... Remove this comment to see the full error message */}
+                    <th scope="row">{GNOMAD_POPULATION_NAMES[popId]}</th>
+                    <td>{(sampleCounts.gnomad_r3.genomes[popId] || 0).toLocaleString()}</td>
```
Comment (phildarnowsky-broad) [inline browser/help/faq/technical-details/what-genetic-ancestry-groups-are-represented-in-the-gnomad-data.tsx:65]: I'd rather do this with some proper iteration and so forth but at this point I'm OK with fixing that in post.

**Review by phildarnowsky-broad — APPROVED** [review]
So let it be written, so let it be merged.

**Pull request merged**